ocaml/xapi-cli-server/cli_frontend.ml

```diff
@@ -3091,6 +3091,17 @@ let rec cmdtable_data : (string * cmd_spec) list =
       ; flags= []
       }
     )
+  ; ( "pool-sync-bundle"
```

Comment:
From user point of view, I think it is used to "upload" the bundle, while "unpack" and ”sync" are our internal operations on the uploaded bundle, then how about rename it to `pool-upload-bundle`?

Reply:
I think we have a `pool-sync-updates` already. So `pool-sync-bundle` is being proposed here.